### ISSUE TITLE
Implement role: Cupid

### ISSUE THREAD

**Issue by AshleyPinner** [chrisgillis/slackwolf#12]
Unsure if you want to implement roles from similar werewolves type games (such as [Werewolves of Miller's Hollow - the village](https://boardgamegeek.com/boardgame/56885/werewolves-millers-hollow-village)), but Cupid is a fun role to have in the game.

If in doubt, this is the rules of Cupid/Lovers:

Cupid wakes up during the setup night phase and chooses two people to become Lovers. Cupid returns to sleep Lovers wake up, learn who the other is, and are told each others roles. 

For the Lovers to win, they need to be the only two left alive; otherwise they win with the village if the village wins. Lovers are unable to vote for each other, and they're not allowed to declare that they're Lovers (well, you can let them, but then they're easy pickings for the wolves).

If a Lover dies, the other Lover dies of heartbreak.

**Comment by chrisgillis** [comment]
Yup, I've been picking roles from Ultimate Werewolf but the Lovers are present in most ww variants / offshoots. Never played with them because I always thought they need a high player count to have a chance at winning. What do you think the minimum number of players would be for Lovers to be included? 10-12+ i'm thinking?

**Comment by AshleyPinner** [comment]
Depends. If they're both human, then you end up with 2 humans that know each other are human (this is akin to playing the Twin Sisters). In that case, you want at least 8 _villagers/villager friendly roles_, but you can always balance the lovers by adding another werewolf. 

If the lovers end up being wolf/human combo, then it's not an advantage to either group, so you can get away with that with lower numbers.
